standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/conf/MetastoreConf.java

```diff
@@ -1826,29 +1826,29 @@ public enum ConfVars {
             new StringSetValidator("simple", "jwt"),
         "Property-maps servlet authentication method (simple or jwt)."
     ),
-    ICEBERG_CATALOG_SERVLET_FACTORY("hive.metastore.catalog.servlet.factory",
-            "hive.metastore.catalog.servlet.factory",
+    ICEBERG_CATALOG_SERVLET_FACTORY("metastore.iceberg.catalog.servlet.factory",
```

Comment:
I removed the "hive." prefix, assuming the metastore related parameter should start with "metastore.". I will revert it if I am wrong

Reply:
I am not sure whether this change is backward compatible.

Reply:
This feature is quite new; We've not cut any release tag since we merged the original change

Reply:
Ok, but won't there be downstream users who have more frequent releases than Apache Hive and which have these configs already in their production environment. Anyways, I am not sure as well whether we should block this since Apache Hive is independent of such releases.
cc @deniskuzZ Do you think this is a good idea?

Reply:
I personally think they use a snapshot version of Hive on their responsibility. Additionally, this feature doesn't work for some reason yet. In my env, NoSuchMethodError happens because of the dependency hell, and some clients can't access tables unless we support Iceberg views(our HiveCatalog doesn't support it).